applications/GeoMechanicsApplication/custom_conditions/Pw_condition.cpp

```diff
@@ -133,6 +133,12 @@ Condition::DofsVectorType PwCondition<TDim, TNumNodes>::GetDofs() const
     return Geo::DofUtilities::ExtractDofsFromNodes(GetGeometry(), WATER_PRESSURE);
 }
 
+template <unsigned int TDim, unsigned int TNumNodes>
+std::string PwCondition<TDim, TNumNodes>::Info() const
```

Comment:
One more general question: This is now the info function, what is the different with `Name()` and might it be more suitable to use that one instead of info if we only add the name?

Reply:
We have found that `Name()` is not a member of any condition class (but it is a member of class template `Geometry`).